### PR TITLE
[Blog] Week 8, 2020

### PR DESCRIPTION
**Pull request by sakalosj**

### CODE REVIEW

**Comment by sakalosj** [comment]
> See the other blog-posts (e.g. https://packit.dev/posts/february-2020/); they are in the form of sentences so we should be consistent.
> 
> (And maybe show only user-related changes to not confuse / bore them..;)

fixed, please check

File: content/posts/February-2020.md [hunk 1]
```diff
@@ -62,3 +62,25 @@ PostgreSQL as a data backend.
...
 [report a failure]: https://github.com/packit-service/packit-service/pull/399
 [serializable]: https://github.com/packit-service/packit-service/pull/391
+
+## Week 8
+
+Postgresql database was introduced in production environment to improve performance and enable implementation 
+of new features, e.g. storing logs to reduce amount of messages sent directly to pull request [(#406)], [(#420)].
+
+Cleanup in configuration files was performed, so no longer needed values were removed from .packit.yaml [(#709)]. User experience was improved 
+by adding new option for command line interface `--upstrem-ref` to copr-build [(#718)] and making error reporting more robust by adding
```
Comment (jpopelka) [inline content/posts/February-2020.md]:
```suggestion
by adding new `--upstrem-ref` option to `copr-build` command in command-line interface [(#718)] and making error reporting more robust by adding 
```

File: content/posts/February-2020.md [hunk 2]
```diff
@@ -62,3 +62,25 @@ PostgreSQL as a data backend.
...
+Postgresql database was introduced in production environment to improve performance and enable implementation 
+of new features, e.g. storing logs to reduce amount of messages sent directly to pull request [(#406)], [(#420)].
+
+Cleanup in configuration files was performed, so no longer needed values were removed from .packit.yaml [(#709)]. User experience was improved 
+by adding new option for command line interface `--upstrem-ref` to copr-build [(#718)] and making error reporting more robust by adding 
+new fedpkg clones related error message [(#714)]. 
+
+Following bugs were fixed:
+- bug in copr-build command fixed [(#713)])
+- get_local_package_config()  duplicate entries in 'directories' bug fixed([#715](https://github.com/packit-service/packit/pull/715))
```
Comment (jpopelka) [inline content/posts/February-2020.md]:
```suggestion
- get_local_package_config() duplicate entries in 'directories' bug fixed ([#715](https://github.com/packit-service/packit/pull/715))
```

File: content/posts/February-2020.md (hunk 1 above)
Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

File: content/posts/February-2020.md (hunk 2 above)
Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

**Pull request merged**